src/lib/packlib.c

```diff
@@ -65,22 +65,20 @@ PWOpen(prefix, mode)
     char *mode;
 {
     int use64 = 0;
-    static PWDICT pdesc;
-    static PWDICT64 pdesc64;
+    PWDICT *pdesc;
+    PWDICT64 pdesc64;
```

Comment:
This is the only piece of this suggested PR that sticks out as questionable - would prefer to see the handling of the dictionaries kept consistent - and not to where the 32 bit one is different. 

There is also some potential overlap with #80 so this may be moot concern.